zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -693,6 +705,12 @@ def list_postgres_databases(*args, **kwargs):
         return []
 
 
+def is_postgresql_operator_present():
+    pg = PostgreSQLClient()
```

Comment:
Is constructor doing some heavylifting job like reading configs from somewhere? If so, can we avoid it by caching `is_operator_present` property (if it's static and does not change in runtime, of course)

Reply:
No, it just sets a couple of attributes.